### DISCUSSION TITLE
HELP!!!! How do i stop this?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1286] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I had to factory reset my chromebook and when i rebooted i was still on 127 but i forgot to caub and now it says restart to update if i put a caub in will it still update and how do i stop it?

**Comment by prjamming** [comment, marked as answer]
Just don't reboot. CAUB won't stop it because the update has already been downloaded. CAUB stops the download.

**Comment by akabutnicer** [comment]
i mean to put it most blatantly and directly, it's over for you; the admins made you their little boy. 💀

**Comment by 88411** [comment]
Happened to me a while back. There really isn't anything you can do other than downgrading. What I tried when I got the message was power washing in hopes that it would wipe the update. It didn't work lol.
